mira/dkg/metaregistry/utils.py

```diff
@@ -43,4 +43,7 @@ def get_app(
         config = parse_config(config)
 
     manager = Manager(registry=config.registry, collections=config.collections, contexts={})
-    return bioregistry.app.impl.get_app(manager=manager, config=config.web)
+    flask_app = bioregistry.app.impl.get_app(manager=manager, config=config.web)
+    if root_prefix:
+        flask_app.config["APPLICATION_ROOT"] = root_prefix
```

Comment:
since we're already generating a generic configuration file that gets put into the flask app's config, should we just update how that configuration file is generated to include this?

Reply:
e.g., we could add `APPLICATION_ROOT` as a key in https://github.com/indralab/mira/blob/main/mira/dkg/metaregistry/epi.json. But I guess I see the value to having this part configurable from docker since this is even more meta-configuration